oggopusEncoder.js

```diff
@@ -78,7 +78,9 @@ OggOpusEncoder.prototype.encode = function( buffers ) {
 };
 
 OggOpusEncoder.prototype.encodeFinalFrame = function() {
-  this.encode( new Float32Array( this.encoderBufferLength - this.encoderBufferIndex ) );
+  this.encode(new Array(this.numberOfChannels).fill().map(function() {
+    return Float32Array(this.encoderBufferLength - this.encoderBufferIndex);
```

Comment:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/fill
Failed to spot the part says it requires a flag to work on Chrome. I'll go with the old way.